docs/agents/sematext-agent/events.md

```diff
@@ -0,0 +1,63 @@
+title: Sematext Agents Events
+
+Below you can find a list of all the events sent from Sematext Agent
+
+## Agent started
+
+This event is sent when the agent is started.
+
+Message: agent_started sematext-agent `version` / on `hostname`
+
+## Agent stopped
+
+This event is sent when the agent is stopped.
+
+Message: agent_stopped sematext-agent `version` / on `hostname`
+
+## Out of memory (OOM)
+
+This event is sent when a process is terminated by OOM Killer.
+
+Message: `process name` process with pid `pid` terminated by OOM killer. Total allocated memory `N` bytes
+
+## Package installed 
+
+This event is sent when a new package is installed (Linux only).
+
+Message: Package `package name`, version `version` has been installed on host `hostname`
+
+Package types supported: Node, Python 2.x/3.x, Deb, RPM
+
+## Package removed
+
+This event is sent when a new package is removed (Linux only).
+
+Message: Package `package name`, version `version` has been removed on host `hostname`
+
+Package types supported: Node, Python 2.x/3.x, Deb, RPM
+
+## Docker events
```

Comment:
@yago-123 Docker or Container?
@cpipilas we seem to be using Docker and Container interchangeably.  Tag names are container.xxxx.

Reply:
Add Container events section and Docker events subsection